src/view/com/util/post-embeds/VideoEmbedInner.web.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect, useRef} from 'react'
+import Hls from 'hls.js'
+
+import {atoms as a} from '#/alf'
+
+export const VideoEmbedInner = ({source}: {source: string}) => {
```

Comment:
not reviewing this because i think we address it on part 2, correct me if wrong tho

Reply:
Correct!